[2024-08-13T02:58:52.621+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: process_etl_heart_data.normalize_numerical_features scheduled__2024-08-12T00:00:00+00:00 [queued]>
[2024-08-13T02:58:52.624+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: process_etl_heart_data.normalize_numerical_features scheduled__2024-08-12T00:00:00+00:00 [queued]>
[2024-08-13T02:58:52.624+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-08-13T02:58:52.629+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonVirtualenvDecoratedOperator): normalize_numerical_features> on 2024-08-12 00:00:00+00:00
[2024-08-13T02:58:52.631+0000] {standard_task_runner.py:60} INFO - Started process 716 to run task
[2024-08-13T02:58:52.633+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'process_etl_heart_data', 'normalize_numerical_features', 'scheduled__2024-08-12T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/etl_process.py', '--cfg-path', '/tmp/tmp5y1a2f1b']
[2024-08-13T02:58:52.634+0000] {standard_task_runner.py:88} INFO - Job 5: Subtask normalize_numerical_features
[2024-08-13T02:58:52.653+0000] {task_command.py:423} INFO - Running <TaskInstance: process_etl_heart_data.normalize_numerical_features scheduled__2024-08-12T00:00:00+00:00 [running]> on host 47ccda1db683
[2024-08-13T02:58:52.680+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Facundo Adrian Lucianna' AIRFLOW_CTX_DAG_ID='process_etl_heart_data' AIRFLOW_CTX_TASK_ID='normalize_numerical_features' AIRFLOW_CTX_EXECUTION_DATE='2024-08-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-12T00:00:00+00:00'
[2024-08-13T02:58:52.681+0000] {process_utils.py:182} INFO - Executing cmd: /usr/local/bin/python -m virtualenv /tmp/venv4zi8wsqd --system-site-packages --python=python
[2024-08-13T02:58:52.683+0000] {process_utils.py:186} INFO - Output:
[2024-08-13T02:58:52.963+0000] {process_utils.py:190} INFO - created virtual environment CPython3.8.18.final.0-64 in 127ms
[2024-08-13T02:58:52.963+0000] {process_utils.py:190} INFO -   creator CPython3Posix(dest=/tmp/venv4zi8wsqd, clear=False, no_vcs_ignore=False, global=True)
[2024-08-13T02:58:52.964+0000] {process_utils.py:190} INFO -   seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/tmp/tmpt0scoh9k)
[2024-08-13T02:58:52.964+0000] {process_utils.py:190} INFO -     added seed packages: pip==23.3.1, setuptools==69.0.2, wheel==0.42.0
[2024-08-13T02:58:52.964+0000] {process_utils.py:190} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2024-08-13T02:58:52.978+0000] {process_utils.py:182} INFO - Executing cmd: /tmp/venv4zi8wsqd/bin/pip install -r /tmp/venv4zi8wsqd/requirements.txt
[2024-08-13T02:58:52.980+0000] {process_utils.py:186} INFO - Output:
[2024-08-13T02:58:53.342+0000] {process_utils.py:190} INFO - WARNING: The directory '/home/***/.cache/pip' or its parent directory is not owned or is not writable by the current user. The cache has been disabled. Check the permissions and owner of that directory. If executing pip with sudo, you should use sudo's -H flag.
[2024-08-13T02:58:53.486+0000] {process_utils.py:190} INFO - Collecting awswrangler==3.6.0 (from -r /tmp/venv4zi8wsqd/requirements.txt (line 1))
[2024-08-13T02:58:53.534+0000] {process_utils.py:190} INFO -   Downloading awswrangler-3.6.0-py3-none-any.whl.metadata (23 kB)
[2024-08-13T02:58:53.712+0000] {process_utils.py:190} INFO - Collecting mlflow==2.10.2 (from -r /tmp/venv4zi8wsqd/requirements.txt (line 2))
[2024-08-13T02:58:53.726+0000] {process_utils.py:190} INFO -   Downloading mlflow-2.10.2-py3-none-any.whl.metadata (13 kB)
[2024-08-13T02:58:53.727+0000] {process_utils.py:190} INFO - Requirement already satisfied: scikit-learn==1.3.2 in /home/***/.local/lib/python3.8/site-packages (from -r /tmp/venv4zi8wsqd/requirements.txt (line 3)) (1.3.2)
[2024-08-13T02:58:53.750+0000] {process_utils.py:190} INFO - Requirement already satisfied: boto3<2.0.0,>=1.20.32 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venv4zi8wsqd/requirements.txt (line 1)) (1.34.158)
[2024-08-13T02:58:53.750+0000] {process_utils.py:190} INFO - Requirement already satisfied: botocore<2.0.0,>=1.23.32 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venv4zi8wsqd/requirements.txt (line 1)) (1.34.158)
[2024-08-13T02:58:53.751+0000] {process_utils.py:190} INFO - Requirement already satisfied: numpy<2.0,>=1.18 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venv4zi8wsqd/requirements.txt (line 1)) (1.24.4)
[2024-08-13T02:58:53.751+0000] {process_utils.py:190} INFO - Requirement already satisfied: packaging<24.0,>=21.1 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venv4zi8wsqd/requirements.txt (line 1)) (23.2)
[2024-08-13T02:58:53.751+0000] {process_utils.py:190} INFO - Requirement already satisfied: pandas<2.1.0,>=1.2.0 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venv4zi8wsqd/requirements.txt (line 1)) (2.0.3)
[2024-08-13T02:58:53.752+0000] {process_utils.py:190} INFO - Requirement already satisfied: pyarrow>=7.0.0 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venv4zi8wsqd/requirements.txt (line 1)) (14.0.2)
[2024-08-13T02:58:53.752+0000] {process_utils.py:190} INFO - Requirement already satisfied: typing-extensions<5.0.0,>=4.4.0 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venv4zi8wsqd/requirements.txt (line 1)) (4.12.2)
[2024-08-13T02:58:53.776+0000] {process_utils.py:190} INFO - Requirement already satisfied: click<9,>=7.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (8.1.7)
[2024-08-13T02:58:53.776+0000] {process_utils.py:190} INFO - Requirement already satisfied: cloudpickle<4 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (3.0.0)
[2024-08-13T02:58:53.777+0000] {process_utils.py:190} INFO - Requirement already satisfied: entrypoints<1 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (0.4)
[2024-08-13T02:58:53.777+0000] {process_utils.py:190} INFO - Requirement already satisfied: gitpython<4,>=2.1.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (3.1.43)
[2024-08-13T02:58:53.778+0000] {process_utils.py:190} INFO - Requirement already satisfied: pyyaml<7,>=5.1 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (6.0.1)
[2024-08-13T02:58:53.778+0000] {process_utils.py:190} INFO - Requirement already satisfied: protobuf<5,>=3.12.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (4.25.2)
[2024-08-13T02:58:53.778+0000] {process_utils.py:190} INFO - Requirement already satisfied: pytz<2024 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (2023.3.post1)
[2024-08-13T02:58:53.779+0000] {process_utils.py:190} INFO - Requirement already satisfied: requests<3,>=2.17.3 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (2.31.0)
[2024-08-13T02:58:53.779+0000] {process_utils.py:190} INFO - Requirement already satisfied: importlib-metadata!=4.7.0,<8,>=3.7.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (6.11.0)
[2024-08-13T02:58:53.779+0000] {process_utils.py:190} INFO - Requirement already satisfied: sqlparse<1,>=0.4.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (0.4.4)
[2024-08-13T02:58:53.780+0000] {process_utils.py:190} INFO - Requirement already satisfied: alembic!=1.10.0,<2 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (1.13.1)
[2024-08-13T02:58:53.780+0000] {process_utils.py:190} INFO - Requirement already satisfied: docker<8,>=4.0.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (7.0.0)
[2024-08-13T02:58:53.780+0000] {process_utils.py:190} INFO - Requirement already satisfied: Flask<4 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (2.2.5)
[2024-08-13T02:58:53.781+0000] {process_utils.py:190} INFO - Requirement already satisfied: scipy<2 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (1.10.1)
[2024-08-13T02:58:53.781+0000] {process_utils.py:190} INFO - Requirement already satisfied: querystring-parser<2 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (1.2.4)
[2024-08-13T02:58:53.781+0000] {process_utils.py:190} INFO - Requirement already satisfied: sqlalchemy<3,>=1.4.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (1.4.51)
[2024-08-13T02:58:53.782+0000] {process_utils.py:190} INFO - Requirement already satisfied: markdown<4,>=3.3 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (3.5.2)
[2024-08-13T02:58:53.782+0000] {process_utils.py:190} INFO - Requirement already satisfied: matplotlib<4 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (3.7.5)
[2024-08-13T02:58:53.783+0000] {process_utils.py:190} INFO - Requirement already satisfied: gunicorn<22 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (21.2.0)
[2024-08-13T02:58:53.783+0000] {process_utils.py:190} INFO - Requirement already satisfied: Jinja2<4,>=2.11 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (3.1.3)
[2024-08-13T02:58:53.797+0000] {process_utils.py:190} INFO - Requirement already satisfied: joblib>=1.1.1 in /home/***/.local/lib/python3.8/site-packages (from scikit-learn==1.3.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 3)) (1.4.2)
[2024-08-13T02:58:53.797+0000] {process_utils.py:190} INFO - Requirement already satisfied: threadpoolctl>=2.0.0 in /home/***/.local/lib/python3.8/site-packages (from scikit-learn==1.3.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 3)) (3.5.0)
[2024-08-13T02:58:53.800+0000] {process_utils.py:190} INFO - Requirement already satisfied: Mako in /home/***/.local/lib/python3.8/site-packages (from alembic!=1.10.0,<2->mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (1.3.0)
[2024-08-13T02:58:53.801+0000] {process_utils.py:190} INFO - Requirement already satisfied: importlib-resources in /home/***/.local/lib/python3.8/site-packages (from alembic!=1.10.0,<2->mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (5.13.0)
[2024-08-13T02:58:53.803+0000] {process_utils.py:190} INFO - Requirement already satisfied: jmespath<2.0.0,>=0.7.1 in /home/***/.local/lib/python3.8/site-packages (from boto3<2.0.0,>=1.20.32->awswrangler==3.6.0->-r /tmp/venv4zi8wsqd/requirements.txt (line 1)) (0.10.0)
[2024-08-13T02:58:53.803+0000] {process_utils.py:190} INFO - Requirement already satisfied: s3transfer<0.11.0,>=0.10.0 in /home/***/.local/lib/python3.8/site-packages (from boto3<2.0.0,>=1.20.32->awswrangler==3.6.0->-r /tmp/venv4zi8wsqd/requirements.txt (line 1)) (0.10.2)
[2024-08-13T02:58:53.806+0000] {process_utils.py:190} INFO - Requirement already satisfied: python-dateutil<3.0.0,>=2.1 in /home/***/.local/lib/python3.8/site-packages (from botocore<2.0.0,>=1.23.32->awswrangler==3.6.0->-r /tmp/venv4zi8wsqd/requirements.txt (line 1)) (2.8.2)
[2024-08-13T02:58:53.807+0000] {process_utils.py:190} INFO - Requirement already satisfied: urllib3<1.27,>=1.25.4 in /home/***/.local/lib/python3.8/site-packages (from botocore<2.0.0,>=1.23.32->awswrangler==3.6.0->-r /tmp/venv4zi8wsqd/requirements.txt (line 1)) (1.26.18)
[2024-08-13T02:58:53.815+0000] {process_utils.py:190} INFO - Requirement already satisfied: Werkzeug>=2.2.2 in /home/***/.local/lib/python3.8/site-packages (from Flask<4->mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (2.2.3)
[2024-08-13T02:58:53.816+0000] {process_utils.py:190} INFO - Requirement already satisfied: itsdangerous>=2.0 in /home/***/.local/lib/python3.8/site-packages (from Flask<4->mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (2.1.2)
[2024-08-13T02:58:53.824+0000] {process_utils.py:190} INFO - Requirement already satisfied: gitdb<5,>=4.0.1 in /home/***/.local/lib/python3.8/site-packages (from gitpython<4,>=2.1.0->mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (4.0.11)
[2024-08-13T02:58:53.843+0000] {process_utils.py:190} INFO - Requirement already satisfied: zipp>=0.5 in /home/***/.local/lib/python3.8/site-packages (from importlib-metadata!=4.7.0,<8,>=3.7.0->mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (3.17.0)
[2024-08-13T02:58:53.845+0000] {process_utils.py:190} INFO - Requirement already satisfied: MarkupSafe>=2.0 in /home/***/.local/lib/python3.8/site-packages (from Jinja2<4,>=2.11->mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (2.1.3)
[2024-08-13T02:58:53.855+0000] {process_utils.py:190} INFO - Requirement already satisfied: contourpy>=1.0.1 in /home/***/.local/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (1.1.1)
[2024-08-13T02:58:53.855+0000] {process_utils.py:190} INFO - Requirement already satisfied: cycler>=0.10 in /home/***/.local/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (0.12.1)
[2024-08-13T02:58:53.856+0000] {process_utils.py:190} INFO - Requirement already satisfied: fonttools>=4.22.0 in /home/***/.local/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (4.53.1)
[2024-08-13T02:58:53.856+0000] {process_utils.py:190} INFO - Requirement already satisfied: kiwisolver>=1.0.1 in /home/***/.local/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (1.4.5)
[2024-08-13T02:58:53.856+0000] {process_utils.py:190} INFO - Requirement already satisfied: pillow>=6.2.0 in /home/***/.local/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (10.4.0)
[2024-08-13T02:58:53.857+0000] {process_utils.py:190} INFO - Requirement already satisfied: pyparsing>=2.3.1 in /home/***/.local/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (3.1.1)
[2024-08-13T02:58:53.895+0000] {process_utils.py:190} INFO - Requirement already satisfied: tzdata>=2022.1 in /home/***/.local/lib/python3.8/site-packages (from pandas<2.1.0,>=1.2.0->awswrangler==3.6.0->-r /tmp/venv4zi8wsqd/requirements.txt (line 1)) (2023.4)
[2024-08-13T02:58:53.903+0000] {process_utils.py:190} INFO - Requirement already satisfied: six in /home/***/.local/lib/python3.8/site-packages (from querystring-parser<2->mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (1.16.0)
[2024-08-13T02:58:53.907+0000] {process_utils.py:190} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests<3,>=2.17.3->mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (3.3.2)
[2024-08-13T02:58:53.907+0000] {process_utils.py:190} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests<3,>=2.17.3->mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (3.6)
[2024-08-13T02:58:53.907+0000] {process_utils.py:190} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests<3,>=2.17.3->mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (2023.11.17)
[2024-08-13T02:58:53.948+0000] {process_utils.py:190} INFO - Requirement already satisfied: greenlet!=0.4.17 in /home/***/.local/lib/python3.8/site-packages (from sqlalchemy<3,>=1.4.0->mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (3.0.3)
[2024-08-13T02:58:53.994+0000] {process_utils.py:190} INFO - Requirement already satisfied: smmap<6,>=3.0.1 in /home/***/.local/lib/python3.8/site-packages (from gitdb<5,>=4.0.1->gitpython<4,>=2.1.0->mlflow==2.10.2->-r /tmp/venv4zi8wsqd/requirements.txt (line 2)) (5.0.1)
[2024-08-13T02:58:54.084+0000] {process_utils.py:190} INFO - Downloading awswrangler-3.6.0-py3-none-any.whl (403 kB)
[2024-08-13T02:58:54.117+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 403.8/403.8 kB 12.5 MB/s eta 0:00:00
[2024-08-13T02:58:54.758+0000] {process_utils.py:190} INFO - Downloading mlflow-2.10.2-py3-none-any.whl (19.5 MB)
[2024-08-13T02:58:57.519+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 19.5/19.5 MB 5.9 MB/s eta 0:00:00
[2024-08-13T02:58:58.602+0000] {process_utils.py:190} INFO - Installing collected packages: mlflow, awswrangler
[2024-08-13T02:58:58.603+0000] {process_utils.py:190} INFO -   Attempting uninstall: mlflow
[2024-08-13T02:58:58.603+0000] {process_utils.py:190} INFO -     Found existing installation: mlflow 2.15.1
[2024-08-13T02:58:58.604+0000] {process_utils.py:190} INFO -     Not uninstalling mlflow at /home/***/.local/lib/python3.8/site-packages, outside environment /tmp/venv4zi8wsqd
[2024-08-13T02:58:58.604+0000] {process_utils.py:190} INFO -     Can't uninstall 'mlflow'. No files were found to uninstall.
[2024-08-13T02:58:59.135+0000] {process_utils.py:190} INFO - Successfully installed awswrangler-3.6.0 mlflow-2.10.2
[2024-08-13T02:58:59.209+0000] {process_utils.py:190} INFO - 
[2024-08-13T02:58:59.209+0000] {process_utils.py:190} INFO - [notice] A new release of pip is available: 23.3.1 -> 24.2
[2024-08-13T02:58:59.209+0000] {process_utils.py:190} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2024-08-13T02:58:59.267+0000] {process_utils.py:182} INFO - Executing cmd: /tmp/venv4zi8wsqd/bin/python /tmp/venv-calli60ddxi7/script.py /tmp/venv-calli60ddxi7/script.in /tmp/venv-calli60ddxi7/script.out /tmp/venv-calli60ddxi7/string_args.txt /tmp/venv-calli60ddxi7/termination.log
[2024-08-13T02:58:59.269+0000] {process_utils.py:186} INFO - Output:
[2024-08-13T02:59:00.288+0000] {process_utils.py:190} INFO - [[34m2024-08-13T02:59:00.287+0000[0m] {[34mcredentials.py:[0m1147} INFO[0m - Found credentials in environment variables.[0m
[2024-08-13T02:59:00.314+0000] {process_utils.py:190} INFO - [[34m2024-08-13T02:59:00.314+0000[0m] {[34mconfigprovider.py:[0m952} INFO[0m - Found endpoint for s3 via: environment_service.[0m
[2024-08-13T02:59:00.332+0000] {process_utils.py:190} INFO - [[34m2024-08-13T02:59:00.332+0000[0m] {[34mcredentials.py:[0m1147} INFO[0m - Found credentials in environment variables.[0m
[2024-08-13T02:59:00.398+0000] {process_utils.py:190} INFO - [[34m2024-08-13T02:59:00.398+0000[0m] {[34mconfigprovider.py:[0m952} INFO[0m - Found endpoint for s3 via: environment_service.[0m
[2024-08-13T02:59:00.417+0000] {process_utils.py:190} INFO - [[34m2024-08-13T02:59:00.417+0000[0m] {[34mcredentials.py:[0m1147} INFO[0m - Found credentials in environment variables.[0m
[2024-08-13T02:59:00.441+0000] {process_utils.py:190} INFO - [[34m2024-08-13T02:59:00.441+0000[0m] {[34mconfigprovider.py:[0m952} INFO[0m - Found endpoint for s3 via: environment_service.[0m
[2024-08-13T02:59:00.459+0000] {process_utils.py:190} INFO - [[34m2024-08-13T02:59:00.458+0000[0m] {[34mcredentials.py:[0m1147} INFO[0m - Found credentials in environment variables.[0m
[2024-08-13T02:59:00.487+0000] {process_utils.py:190} INFO - [[34m2024-08-13T02:59:00.487+0000[0m] {[34mconfigprovider.py:[0m952} INFO[0m - Found endpoint for s3 via: environment_service.[0m
[2024-08-13T02:59:00.500+0000] {process_utils.py:190} INFO - [[34m2024-08-13T02:59:00.499+0000[0m] {[34mcredentials.py:[0m1147} INFO[0m - Found credentials in environment variables.[0m
[2024-08-13T02:59:00.526+0000] {process_utils.py:190} INFO - [[34m2024-08-13T02:59:00.526+0000[0m] {[34mconfigprovider.py:[0m952} INFO[0m - Found endpoint for s3 via: environment_service.[0m
[2024-08-13T02:59:00.835+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-08-13T02:59:00.841+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=process_etl_heart_data, task_id=normalize_numerical_features, execution_date=20240812T000000, start_date=20240813T025852, end_date=20240813T025900
[2024-08-13T02:59:00.894+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-08-13T02:59:00.902+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-13T23:06:50.845+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: process_etl_heart_data.normalize_numerical_features scheduled__2024-08-12T00:00:00+00:00 [queued]>
[2024-08-13T23:06:50.849+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: process_etl_heart_data.normalize_numerical_features scheduled__2024-08-12T00:00:00+00:00 [queued]>
[2024-08-13T23:06:50.849+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-08-13T23:06:50.854+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonVirtualenvDecoratedOperator): normalize_numerical_features> on 2024-08-12 00:00:00+00:00
[2024-08-13T23:06:50.857+0000] {standard_task_runner.py:60} INFO - Started process 879 to run task
[2024-08-13T23:06:50.858+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'process_etl_heart_data', 'normalize_numerical_features', 'scheduled__2024-08-12T00:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/etl_process.py', '--cfg-path', '/tmp/tmp5s2_ogrl']
[2024-08-13T23:06:50.859+0000] {standard_task_runner.py:88} INFO - Job 8: Subtask normalize_numerical_features
[2024-08-13T23:06:50.880+0000] {task_command.py:423} INFO - Running <TaskInstance: process_etl_heart_data.normalize_numerical_features scheduled__2024-08-12T00:00:00+00:00 [running]> on host 724c87fe0c82
[2024-08-13T23:06:50.913+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Facundo Adrian Lucianna' AIRFLOW_CTX_DAG_ID='process_etl_heart_data' AIRFLOW_CTX_TASK_ID='normalize_numerical_features' AIRFLOW_CTX_EXECUTION_DATE='2024-08-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-12T00:00:00+00:00'
[2024-08-13T23:06:50.914+0000] {process_utils.py:182} INFO - Executing cmd: /usr/local/bin/python -m virtualenv /tmp/venvpw4jq0ws --system-site-packages --python=python
[2024-08-13T23:06:50.916+0000] {process_utils.py:186} INFO - Output:
[2024-08-13T23:06:51.161+0000] {process_utils.py:190} INFO - created virtual environment CPython3.8.18.final.0-64 in 80ms
[2024-08-13T23:06:51.165+0000] {process_utils.py:190} INFO -   creator CPython3Posix(dest=/tmp/venvpw4jq0ws, clear=False, no_vcs_ignore=False, global=True)
[2024-08-13T23:06:51.165+0000] {process_utils.py:190} INFO -   seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2024-08-13T23:06:51.165+0000] {process_utils.py:190} INFO -     added seed packages: pip==23.3.1, setuptools==69.0.2, wheel==0.42.0
[2024-08-13T23:06:51.165+0000] {process_utils.py:190} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2024-08-13T23:06:51.181+0000] {process_utils.py:182} INFO - Executing cmd: /tmp/venvpw4jq0ws/bin/pip install -r /tmp/venvpw4jq0ws/requirements.txt
[2024-08-13T23:06:51.184+0000] {process_utils.py:186} INFO - Output:
[2024-08-13T23:06:51.709+0000] {process_utils.py:190} INFO - Collecting awswrangler==3.6.0 (from -r /tmp/venvpw4jq0ws/requirements.txt (line 1))
[2024-08-13T23:06:51.709+0000] {process_utils.py:190} INFO -   Using cached awswrangler-3.6.0-py3-none-any.whl.metadata (23 kB)
[2024-08-13T23:06:51.742+0000] {process_utils.py:190} INFO - Collecting mlflow==2.10.2 (from -r /tmp/venvpw4jq0ws/requirements.txt (line 2))
[2024-08-13T23:06:51.780+0000] {process_utils.py:190} INFO -   Downloading mlflow-2.10.2-py3-none-any.whl.metadata (13 kB)
[2024-08-13T23:06:51.785+0000] {process_utils.py:190} INFO - Requirement already satisfied: scikit-learn==1.3.2 in /home/***/.local/lib/python3.8/site-packages (from -r /tmp/venvpw4jq0ws/requirements.txt (line 3)) (1.3.2)
[2024-08-13T23:06:51.805+0000] {process_utils.py:190} INFO - Requirement already satisfied: boto3<2.0.0,>=1.20.32 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venvpw4jq0ws/requirements.txt (line 1)) (1.34.158)
[2024-08-13T23:06:51.805+0000] {process_utils.py:190} INFO - Requirement already satisfied: botocore<2.0.0,>=1.23.32 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venvpw4jq0ws/requirements.txt (line 1)) (1.34.158)
[2024-08-13T23:06:51.806+0000] {process_utils.py:190} INFO - Requirement already satisfied: numpy<2.0,>=1.18 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venvpw4jq0ws/requirements.txt (line 1)) (1.24.4)
[2024-08-13T23:06:51.806+0000] {process_utils.py:190} INFO - Requirement already satisfied: packaging<24.0,>=21.1 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venvpw4jq0ws/requirements.txt (line 1)) (23.2)
[2024-08-13T23:06:51.807+0000] {process_utils.py:190} INFO - Requirement already satisfied: pandas<2.1.0,>=1.2.0 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venvpw4jq0ws/requirements.txt (line 1)) (2.0.3)
[2024-08-13T23:06:51.807+0000] {process_utils.py:190} INFO - Requirement already satisfied: pyarrow>=7.0.0 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venvpw4jq0ws/requirements.txt (line 1)) (14.0.2)
[2024-08-13T23:06:51.808+0000] {process_utils.py:190} INFO - Requirement already satisfied: typing-extensions<5.0.0,>=4.4.0 in /home/***/.local/lib/python3.8/site-packages (from awswrangler==3.6.0->-r /tmp/venvpw4jq0ws/requirements.txt (line 1)) (4.12.2)
[2024-08-13T23:06:51.835+0000] {process_utils.py:190} INFO - Requirement already satisfied: click<9,>=7.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (8.1.7)
[2024-08-13T23:06:51.836+0000] {process_utils.py:190} INFO - Requirement already satisfied: cloudpickle<4 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (3.0.0)
[2024-08-13T23:06:51.836+0000] {process_utils.py:190} INFO - Requirement already satisfied: entrypoints<1 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (0.4)
[2024-08-13T23:06:51.837+0000] {process_utils.py:190} INFO - Requirement already satisfied: gitpython<4,>=2.1.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (3.1.43)
[2024-08-13T23:06:51.837+0000] {process_utils.py:190} INFO - Requirement already satisfied: pyyaml<7,>=5.1 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (6.0.1)
[2024-08-13T23:06:51.837+0000] {process_utils.py:190} INFO - Requirement already satisfied: protobuf<5,>=3.12.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (4.25.2)
[2024-08-13T23:06:51.837+0000] {process_utils.py:190} INFO - Requirement already satisfied: pytz<2024 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (2023.3.post1)
[2024-08-13T23:06:51.838+0000] {process_utils.py:190} INFO - Requirement already satisfied: requests<3,>=2.17.3 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (2.31.0)
[2024-08-13T23:06:51.838+0000] {process_utils.py:190} INFO - Requirement already satisfied: importlib-metadata!=4.7.0,<8,>=3.7.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (6.11.0)
[2024-08-13T23:06:51.838+0000] {process_utils.py:190} INFO - Requirement already satisfied: sqlparse<1,>=0.4.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (0.4.4)
[2024-08-13T23:06:51.838+0000] {process_utils.py:190} INFO - Requirement already satisfied: alembic!=1.10.0,<2 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (1.13.1)
[2024-08-13T23:06:51.839+0000] {process_utils.py:190} INFO - Requirement already satisfied: docker<8,>=4.0.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (7.0.0)
[2024-08-13T23:06:51.840+0000] {process_utils.py:190} INFO - Requirement already satisfied: Flask<4 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (2.2.5)
[2024-08-13T23:06:51.840+0000] {process_utils.py:190} INFO - Requirement already satisfied: scipy<2 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (1.10.1)
[2024-08-13T23:06:51.840+0000] {process_utils.py:190} INFO - Requirement already satisfied: querystring-parser<2 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (1.2.4)
[2024-08-13T23:06:51.841+0000] {process_utils.py:190} INFO - Requirement already satisfied: sqlalchemy<3,>=1.4.0 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (1.4.51)
[2024-08-13T23:06:51.841+0000] {process_utils.py:190} INFO - Requirement already satisfied: markdown<4,>=3.3 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (3.5.2)
[2024-08-13T23:06:51.842+0000] {process_utils.py:190} INFO - Requirement already satisfied: matplotlib<4 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (3.7.5)
[2024-08-13T23:06:51.842+0000] {process_utils.py:190} INFO - Requirement already satisfied: gunicorn<22 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (21.2.0)
[2024-08-13T23:06:51.843+0000] {process_utils.py:190} INFO - Requirement already satisfied: Jinja2<4,>=2.11 in /home/***/.local/lib/python3.8/site-packages (from mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (3.1.3)
[2024-08-13T23:06:51.857+0000] {process_utils.py:190} INFO - Requirement already satisfied: joblib>=1.1.1 in /home/***/.local/lib/python3.8/site-packages (from scikit-learn==1.3.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 3)) (1.4.2)
[2024-08-13T23:06:51.857+0000] {process_utils.py:190} INFO - Requirement already satisfied: threadpoolctl>=2.0.0 in /home/***/.local/lib/python3.8/site-packages (from scikit-learn==1.3.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 3)) (3.5.0)
[2024-08-13T23:06:51.861+0000] {process_utils.py:190} INFO - Requirement already satisfied: Mako in /home/***/.local/lib/python3.8/site-packages (from alembic!=1.10.0,<2->mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (1.3.0)
[2024-08-13T23:06:51.861+0000] {process_utils.py:190} INFO - Requirement already satisfied: importlib-resources in /home/***/.local/lib/python3.8/site-packages (from alembic!=1.10.0,<2->mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (5.13.0)
[2024-08-13T23:06:51.864+0000] {process_utils.py:190} INFO - Requirement already satisfied: jmespath<2.0.0,>=0.7.1 in /home/***/.local/lib/python3.8/site-packages (from boto3<2.0.0,>=1.20.32->awswrangler==3.6.0->-r /tmp/venvpw4jq0ws/requirements.txt (line 1)) (0.10.0)
[2024-08-13T23:06:51.865+0000] {process_utils.py:190} INFO - Requirement already satisfied: s3transfer<0.11.0,>=0.10.0 in /home/***/.local/lib/python3.8/site-packages (from boto3<2.0.0,>=1.20.32->awswrangler==3.6.0->-r /tmp/venvpw4jq0ws/requirements.txt (line 1)) (0.10.2)
[2024-08-13T23:06:51.868+0000] {process_utils.py:190} INFO - Requirement already satisfied: python-dateutil<3.0.0,>=2.1 in /home/***/.local/lib/python3.8/site-packages (from botocore<2.0.0,>=1.23.32->awswrangler==3.6.0->-r /tmp/venvpw4jq0ws/requirements.txt (line 1)) (2.8.2)
[2024-08-13T23:06:51.868+0000] {process_utils.py:190} INFO - Requirement already satisfied: urllib3<1.27,>=1.25.4 in /home/***/.local/lib/python3.8/site-packages (from botocore<2.0.0,>=1.23.32->awswrangler==3.6.0->-r /tmp/venvpw4jq0ws/requirements.txt (line 1)) (1.26.18)
[2024-08-13T23:06:51.877+0000] {process_utils.py:190} INFO - Requirement already satisfied: Werkzeug>=2.2.2 in /home/***/.local/lib/python3.8/site-packages (from Flask<4->mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (2.2.3)
[2024-08-13T23:06:51.877+0000] {process_utils.py:190} INFO - Requirement already satisfied: itsdangerous>=2.0 in /home/***/.local/lib/python3.8/site-packages (from Flask<4->mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (2.1.2)
[2024-08-13T23:06:51.886+0000] {process_utils.py:190} INFO - Requirement already satisfied: gitdb<5,>=4.0.1 in /home/***/.local/lib/python3.8/site-packages (from gitpython<4,>=2.1.0->mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (4.0.11)
[2024-08-13T23:06:51.907+0000] {process_utils.py:190} INFO - Requirement already satisfied: zipp>=0.5 in /home/***/.local/lib/python3.8/site-packages (from importlib-metadata!=4.7.0,<8,>=3.7.0->mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (3.17.0)
[2024-08-13T23:06:51.909+0000] {process_utils.py:190} INFO - Requirement already satisfied: MarkupSafe>=2.0 in /home/***/.local/lib/python3.8/site-packages (from Jinja2<4,>=2.11->mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (2.1.3)
[2024-08-13T23:06:51.918+0000] {process_utils.py:190} INFO - Requirement already satisfied: contourpy>=1.0.1 in /home/***/.local/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (1.1.1)
[2024-08-13T23:06:51.919+0000] {process_utils.py:190} INFO - Requirement already satisfied: cycler>=0.10 in /home/***/.local/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (0.12.1)
[2024-08-13T23:06:51.919+0000] {process_utils.py:190} INFO - Requirement already satisfied: fonttools>=4.22.0 in /home/***/.local/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (4.53.1)
[2024-08-13T23:06:51.919+0000] {process_utils.py:190} INFO - Requirement already satisfied: kiwisolver>=1.0.1 in /home/***/.local/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (1.4.5)
[2024-08-13T23:06:51.920+0000] {process_utils.py:190} INFO - Requirement already satisfied: pillow>=6.2.0 in /home/***/.local/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (10.4.0)
[2024-08-13T23:06:51.920+0000] {process_utils.py:190} INFO - Requirement already satisfied: pyparsing>=2.3.1 in /home/***/.local/lib/python3.8/site-packages (from matplotlib<4->mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (3.1.1)
[2024-08-13T23:06:51.959+0000] {process_utils.py:190} INFO - Requirement already satisfied: tzdata>=2022.1 in /home/***/.local/lib/python3.8/site-packages (from pandas<2.1.0,>=1.2.0->awswrangler==3.6.0->-r /tmp/venvpw4jq0ws/requirements.txt (line 1)) (2023.4)
[2024-08-13T23:06:51.968+0000] {process_utils.py:190} INFO - Requirement already satisfied: six in /home/***/.local/lib/python3.8/site-packages (from querystring-parser<2->mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (1.16.0)
[2024-08-13T23:06:51.971+0000] {process_utils.py:190} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests<3,>=2.17.3->mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (3.3.2)
[2024-08-13T23:06:51.972+0000] {process_utils.py:190} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests<3,>=2.17.3->mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (3.6)
[2024-08-13T23:06:51.972+0000] {process_utils.py:190} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests<3,>=2.17.3->mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (2023.11.17)
[2024-08-13T23:06:52.011+0000] {process_utils.py:190} INFO - Requirement already satisfied: greenlet!=0.4.17 in /home/***/.local/lib/python3.8/site-packages (from sqlalchemy<3,>=1.4.0->mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (3.0.3)
[2024-08-13T23:06:52.053+0000] {process_utils.py:190} INFO - Requirement already satisfied: smmap<6,>=3.0.1 in /home/***/.local/lib/python3.8/site-packages (from gitdb<5,>=4.0.1->gitpython<4,>=2.1.0->mlflow==2.10.2->-r /tmp/venvpw4jq0ws/requirements.txt (line 2)) (5.0.1)
[2024-08-13T23:06:52.124+0000] {process_utils.py:190} INFO - Using cached awswrangler-3.6.0-py3-none-any.whl (403 kB)
[2024-08-13T23:06:52.134+0000] {process_utils.py:190} INFO - Downloading mlflow-2.10.2-py3-none-any.whl (19.5 MB)
[2024-08-13T23:06:52.911+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 19.5/19.5 MB 24.0 MB/s eta 0:00:00
[2024-08-13T23:06:54.055+0000] {process_utils.py:190} INFO - Installing collected packages: mlflow, awswrangler
[2024-08-13T23:06:54.055+0000] {process_utils.py:190} INFO -   Attempting uninstall: mlflow
[2024-08-13T23:06:54.056+0000] {process_utils.py:190} INFO -     Found existing installation: mlflow 2.15.1
[2024-08-13T23:06:54.056+0000] {process_utils.py:190} INFO -     Not uninstalling mlflow at /home/***/.local/lib/python3.8/site-packages, outside environment /tmp/venvpw4jq0ws
[2024-08-13T23:06:54.056+0000] {process_utils.py:190} INFO -     Can't uninstall 'mlflow'. No files were found to uninstall.
[2024-08-13T23:06:54.620+0000] {process_utils.py:190} INFO - Successfully installed awswrangler-3.6.0 mlflow-2.10.2
[2024-08-13T23:06:54.689+0000] {process_utils.py:190} INFO - 
[2024-08-13T23:06:54.689+0000] {process_utils.py:190} INFO - [notice] A new release of pip is available: 23.3.1 -> 24.2
[2024-08-13T23:06:54.689+0000] {process_utils.py:190} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2024-08-13T23:06:54.758+0000] {process_utils.py:182} INFO - Executing cmd: /tmp/venvpw4jq0ws/bin/python /tmp/venv-callga_udpv9/script.py /tmp/venv-callga_udpv9/script.in /tmp/venv-callga_udpv9/script.out /tmp/venv-callga_udpv9/string_args.txt /tmp/venv-callga_udpv9/termination.log
[2024-08-13T23:06:54.760+0000] {process_utils.py:186} INFO - Output:
[2024-08-13T23:06:55.881+0000] {process_utils.py:190} INFO - [[34m2024-08-13T23:06:55.880+0000[0m] {[34mcredentials.py:[0m1147} INFO[0m - Found credentials in environment variables.[0m
[2024-08-13T23:06:55.907+0000] {process_utils.py:190} INFO - [[34m2024-08-13T23:06:55.906+0000[0m] {[34mconfigprovider.py:[0m952} INFO[0m - Found endpoint for s3 via: environment_service.[0m
[2024-08-13T23:06:55.926+0000] {process_utils.py:190} INFO - [[34m2024-08-13T23:06:55.926+0000[0m] {[34mcredentials.py:[0m1147} INFO[0m - Found credentials in environment variables.[0m
[2024-08-13T23:06:55.995+0000] {process_utils.py:190} INFO - [[34m2024-08-13T23:06:55.995+0000[0m] {[34mconfigprovider.py:[0m952} INFO[0m - Found endpoint for s3 via: environment_service.[0m
[2024-08-13T23:06:56.016+0000] {process_utils.py:190} INFO - [[34m2024-08-13T23:06:56.016+0000[0m] {[34mcredentials.py:[0m1147} INFO[0m - Found credentials in environment variables.[0m
[2024-08-13T23:06:56.040+0000] {process_utils.py:190} INFO - [[34m2024-08-13T23:06:56.040+0000[0m] {[34mconfigprovider.py:[0m952} INFO[0m - Found endpoint for s3 via: environment_service.[0m
[2024-08-13T23:06:56.054+0000] {process_utils.py:190} INFO - [[34m2024-08-13T23:06:56.054+0000[0m] {[34mcredentials.py:[0m1147} INFO[0m - Found credentials in environment variables.[0m
[2024-08-13T23:06:56.077+0000] {process_utils.py:190} INFO - [[34m2024-08-13T23:06:56.077+0000[0m] {[34mconfigprovider.py:[0m952} INFO[0m - Found endpoint for s3 via: environment_service.[0m
[2024-08-13T23:06:56.089+0000] {process_utils.py:190} INFO - [[34m2024-08-13T23:06:56.089+0000[0m] {[34mcredentials.py:[0m1147} INFO[0m - Found credentials in environment variables.[0m
[2024-08-13T23:06:56.116+0000] {process_utils.py:190} INFO - [[34m2024-08-13T23:06:56.115+0000[0m] {[34mconfigprovider.py:[0m952} INFO[0m - Found endpoint for s3 via: environment_service.[0m
[2024-08-13T23:06:56.431+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-08-13T23:06:56.437+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=process_etl_heart_data, task_id=normalize_numerical_features, execution_date=20240812T000000, start_date=20240813T230650, end_date=20240813T230656
[2024-08-13T23:06:56.466+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-08-13T23:06:56.473+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
